xla/stream_executor/stream.h

```diff
@@ -44,6 +44,10 @@ limitations under the License.
 #include "tsl/platform/errors.h"
 #include "tsl/platform/thread_annotations.h"
 
+namespace cudnn_frontend::graph {
```

Comment:
We should not be depending on cuda/cudnn types in a generic cudnn stream

Reply:
Done.